config/database.yml

```diff
@@ -1,25 +1,12 @@
-# SQLite. Versions 3.8.0 and up are supported.
-#   gem install sqlite3
-#
-#   Ensure the SQLite 3 gem is defined in your Gemfile
-#   gem "sqlite3"
-#
 default: &default
-  adapter: sqlite3
-  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  adapter: postgresql
+  encoding: unicode
+  pool: 5
+  timeout:
 
 development:
   <<: *default
-  database: db/development.sqlite3
-
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
-test:
-  <<: *default
-  database: db/test.sqlite3
-
-production:
-  <<: *default
-  database: db/production.sqlite3
+  database: demon_cinema_development
+  host: localhost
+  username: demon_cinema
```

Comment:
For development & test it's fine. For production we will need to do `ENV["DB_USER"]` and so on.

We don't have to worry about this just yet. We will add `production` when we will deploy to prod for the first time :)

Reply:
I've run into issues with ENVs at the moment (decided to hide the password) but I think I need some help with this whole file :)

Reply:
Sure, no problem - we can cover it on our next session :)